intranet/settings/__init__.py

```diff
@@ -202,7 +204,7 @@
 EMAIL_FROM = "ion-noreply@tjhsst.edu"
 
 # Use PostgreSQL database
-DATABASES = {"default": {"ENGINE": "django_prometheus.db.backends.postgresql", "CONN_MAX_AGE": 30}}  # type: Dict[str,Dict[str,Any]]
+DATABASES: dict[str, dict[str, str | int]] = {"default": {"ENGINE": "django_prometheus.db.backends.postgresql", "CONN_MAX_AGE": 30}}
```

Comment:
like line 560 the type of this should probably be `dict[str, dict[str, Any]]` in case of new keys / values